packages/it-multipart/index.js

```diff
@@ -19,7 +38,7 @@ async function * multipart (stream, boundary) {
   const headerEnd = Buffer.from('\r\n\r\n')
 
   // allow pushing data back into stream
-  stream = prefixStream(stream)
+  const stream = prefixStream(source)
```

Comment:
📝 Type checker would not allow overloading same var here because it's type is `IncomingMessage` which is different from `PrefixPush`.